code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/identity/Token.java

```diff
@@ -0,0 +1,172 @@
+package io.cattle.platform.iaas.api.auth.identity;
+
+        import io.cattle.platform.api.auth.Identity;
+        import io.cattle.platform.iaas.api.auth.TokenUtils;
+        import io.cattle.platform.iaas.api.auth.integration.github.resource.TeamAccountInfo;
+        import io.github.ibuildthecloud.gdapi.annotation.Field;
+        import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+        import java.util.List;
+
+
+@Type(name = TokenUtils.TOKEN)
+public class Token {
+
+    private final String jwt;
+    private final String hostname;
+    private String code;
+    private final String user;
+    private final List<String> orgs;
+    private final List<TeamAccountInfo> teams;
+    private final Boolean security;
+    private final String clientId;
+    private final String userType;
+    private final String authProvider;
+
+    private final String accountId;
+    private final Identity userIdentity;
+    private final boolean enabled;
+    private final List<Identity> identities;
+
+    public Token(String jwt, String username, List<String> orgs, List<TeamAccountInfo> teams, Boolean security, String clientId, String userType,
```

Comment:
Token is made is several places. This one is legacy. Need to mark it as such. It will be replaced by The one lower down.